buildpipeline/pipeline.json

```diff
@@ -279,7 +297,7 @@
             "PB_BuildArguments": "-framework=netfx -buildArch=x64 -Release -- /p:SignType=real /p:RuntimeOS=win10",
             "PB_BuildTestsArguments": "-framework=netfx -buildArch=x64 -Release -SkipTests -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
```

Comment:
Now that you are modifying this, it would be nice if we could enable the outerloop tests for all netfx build definitions.